Samurai/README.md

```diff
@@ -0,0 +1,9 @@
+
+#Samurai
+
+##Screenshots 
+
+![Samurai](./samuraitheme.png)
+
+#more
```

Comment:
Correct the headers and check the preview https://github.com/no3l0815/spicetify-themes/tree/master/Samurai